tap-setup-scripts/src/resources/package_overlay_secrets.yml

```diff
@@ -0,0 +1,30 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#! To add more overlays to packages
+#!   1. add one or more new secrets here
+#!   2. add those secrets to the package_overlay_apply.yml
+
+#@ tapNamespace = data.values.tapNamespace if hasattr(data.values, "tapNamespace") else "tap-install"
+#@ tapVersion = data.values.tapVersion
+#@ quickstartBuild = data.values.quickstartBuild if hasattr(data.values, "quickstartBuild") else "0"
```

Comment:
I don't think that should be a UUID, I think that should be a release version of the QS.
I imagine we'll have the QS eventually versioned and this would be that.

Quoting the story:
```
The key-value pair will be version={TAP_VERSION}+{platform}.{buildID}
Where the 3-character CSP platform abbreviation is used in build metadata.
For instance, for the AWS quickstart running TAP 1.4.0, the string would look like:
version=1.4.0+aws.5
```

For now, I set it to `0` because we essentially don't have proper versioning going yet.